tox.ini

```diff
@@ -1,9 +1,10 @@
 [tox]
-envlist = py27,py33,py34,py35,pypy,pypy3
+envlist = py27,py33,py34,py35,py36,pypy,pypy3
 
 [testenv]
 deps=
     pytest>=2.2.3
     pypy,pypy3: psycopg2cffi>=2.6.1
     py27,py33,py34,py35: psycopg2>=2.4.6
+    py36: psycopg2>=2.4.6
```

Comment:
I'm happy to redo it. I did it like that b/c I don't have the other python interpreters available, so I just commented out that line (and assumed your own ci setup had verified them, since I didn't actually make any changes).